src/Frontend.elm

```diff
@@ -1065,12 +1065,22 @@ formView model productId priceId ticket =
                 , label = el [ centerX ] (text "Cancel")
                 }
 
+        includesAccom =
+            Tickets.formIncludesAccom form
+
+        hasAttendees =
+            List.length form.attendees > 0
+
         orderNotes =
-            if Tickets.formIncludesAccom form && List.length form.attendees == 0 then
-                "<red>Warning: you have chosen accommodation but no attendees, please make sure each attendee has a ticket selection (unless they've purchased them separately).</red>"
+            Debug.log "order notes" <|
```

Comment:
Drop the debug? :)

Reply:
This is why I don't like merging without 1 review 😅